chimba/chain.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "chimba",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://chimba.ooo/",
+  "pretty_name": "Chimba",
+  "chain_id": "chimba",
+  "bech32_prefix": "chimba",
```

Comment:
Please correct the SLIP-173 registration at https://github.com/satoshilabs/slips/blob/master/slip-0173.md . It has 'cmba' as the bech32 prefix, when in fact the prefix is 'chimba'.

Reply:
the change in the file for slip-073 has already been applied